### PR TITLE
Mack Table block

### LINKED ISSUE #122: Table block
Parts and Services

### PR DESCRIPTION
**Pull request by manuel-vara**
Fix #122 

Test URLs:
- Before: https://main--vg-macktrucks-com--hlxsites.hlx.page/drafts/manuel.vara/table/table-block
- After: https://122-table-block--vg-macktrucks-com--hlxsites.hlx.page/drafts/manuel.vara/table/table-block

### CODE REVIEW

File: blocks/table/table.css [hunk 1]
```diff
@@ -55,4 +60,102 @@
...
+    border-collapse: collapse;
+    width: 100%;
+  }
+
+  .block.table.row-header tr {
+    border: 1px solid #d3d3d3;
+  }
+
+  .block.table.row-header td {
+    color: #000;
```
Comment (jonatan-lledo-netcentric) [inline blocks/table/table.css:75]: use _var(--primary-black);_ instead of the color

**Review by jonatan-lledo-netcentric — CHANGES_REQUESTED** [review]
aside for the black colors, it's fine to me

File: blocks/table/table.css (hunk 1 above)
Comment (manuel-vara) [reply blocks/table/table.css]: Update done. Thanks!

File: blocks/table/table.css [hunk 2]
```diff
@@ -55,4 +60,102 @@
...
+  }
+}
+
+@media (max-width: 768px) {
+  .block.table.row-header table {
+    width: 100%;
+  }
+
+  .block.table.row-header tr {
+    border-top: 2px solid #000;
```
[the remark this answers is not in the document]
Comment (manuel-vara) [reply blocks/table/table.css]: Update done. Thanks!

**Review by jonatan-lledo-netcentric — APPROVED** [review]

**Pull request merged**